### PR TITLE
WIP: [zh] sync pod-scheduling-readiness.md

### PR DESCRIPTION
**Pull request by iyear**
Sync with en https://github.com/kubernetes/website/pull/37675

I will proofread again after the original PR is merged.

### CODE REVIEW

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+## 配置 Pod 的调度门控
+
+<!--
+The `schedulingGates` field contains a list of strings, and each string literal is perceived as a
+criteria that Pod should be satisfied before considered schedulable. This field can be initialized
+only when a Pod is created (either by the client, or mutated during admission). After creation,
+each schedulingGate can be removed in arbitrary order, but addition of a new scheduling gate is disallowed.
+-->
+
+`schedulingGates` 字段包含一个字符串列表，每个字符串都为 Pod 在可调度之前应该满足的准则。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md:50]:
```suggestion
`schedulingGates` 字段包含一个字符串列表，每个字符串都被视为 Pod 在可调度之前应该满足的一个条件。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+这个字段只能在 Pod 被创建时（由客户端创建，或在准入过程中突变）初始化。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md:51]:
```suggestion
这个字段只能在 Pod 被创建时（由客户端创建，或在准入过程中变更）初始化。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+在创建之后，每个调度门控可以按任意顺序被删除，但不允许增加新的调度门控。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md:52]:
```suggestion
在创建之后，各个 schedulingGate 可以按任意顺序被删除，但不允许增加新的调度门控。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+if --> s2: 否
+if --> s3: 是
+s3 --> s4
+s4 --> [*]
+{{< /mermaid >}}
+
+<!--
+## Usage example
+-->
+## 使用示例
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md:74]:
```suggestion
## 用法示例    {#usage-example}
```

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+<!--
+To mark a Pod not-ready for scheduling, you can create it with one or more scheduling gates like this:
+
+{{< codenew file="pods/pod-with-scheduling-gates.yaml" >}}
+
+After the Pod's creation, you can check its state using:
+-->
+要标记一个 Pod 未调度就绪，你可以像这样创建它，其中包含一个或多个调度门控：
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md:83]:
```suggestion
要标记一个 Pod 未调度就绪，你可以在创建 Pod 时设置一个或多个调度门控，像这样：
```

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+<!--
+To inform scheduler this Pod is ready for scheduling, you can remove its `schedulingGates` entirely
+by re-applying a modified manifest:
+
+{{< codenew file="pods/pod-without-scheduling-gates.yaml" >}}
+
+You can check if the `schedulingGates` is cleared by running:
+-->
+要通知调度器这个 Pod 已经准备好进行调度，你可以通过重新应用修改过的清单来完全删除其 `schedulingGates` ：
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md:129]:
```suggestion
要通知调度器这个 Pod 已经准备好进行调度，你可以通过重新应用修改过的清单来彻底删除其 `schedulingGates` ：
```

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+## Observability
+-->
+## 可观测性 (Observability)
+
+<!--
+The metric `scheduler_pending_pods` comes with a new label `"gated"` to distinguish whether a Pod
+has been tried scheduling but claimed as unschedulable, or explicitly marked as not ready for
+scheduling. You can use `scheduler_pending_pods{queue="gated"}` to check the metric result.
+-->
+指标 `scheduler_pending_pods` 带有一个新的标签 `"gated"` ，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md:169]: 多一空格

Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md:169]:
```suggestion
指标 `scheduler_pending_pods` 带有一个新的标签 `"gated"`，
```

**Comment by divya-mohan0209** [comment]
Just a quick question, @iyear, I understand that this is the translation for a doc on the 1.26 release. However, is there a requirement/has there been an agreement for this to land in the dev-1.26 branch along with all other English docs? The reason is we typically have the localization teams work on the different docs AFTER the docs for that release are published since updating an isolated doc might result in inconsistencies.

cc : @reylejano @natalisucks @sftim @a-mccarthy

**Comment by iyear** [comment]
@divya-mohan0209 
I'm very sorry for interfering with your work. I just skimmed through the history of zh PRs and contribution docs...

Reasons why I committed to the `dev-1.26` branch: #37981, so I directly submit a new PR...

So I should wait until the new docs are released and I commit the translation to the `dev-1.26-zh.x` branch...?

**Comment by tengqm** [comment]
@iyear The concern from the reviewers is valid. We don't have a lot of localization PRs going into the `dev-1.xy` branch because most localization teams are busy translating current/previous (sub-)versions. The Chinese (zh-cn) localization team, thanks to the passions from all contributors, has been following and closely tracking the `main` branch for some time. We are kind of pushing the limit, contributing to `dev-1.xy` branch directly. This is good.

However, there are some risks on doing this. Given an existing page `en/foo/page.md`, we have a `zh-cn/foo/page.md`. There could be changes to the English version for current release (say '1.25', aka, 'main' branch). These changes can be tracked using the `scripts/lsync.sh` script. There could be NEW changes to the same page in the `dev-1.26` branch. We may want to update the Chinese version in the `dev-1.26` branch as well, but the story is getting complicated. The `en/foo/page.md` now has two versions (one in `main`, one in `dev-1.26`). The release team needs to make sure there are no conflicts between them. This team CANNOT do the same for Chinese localizations. When conflicts do arise, they break the build, or even delay the site release.

So, I'd suggest we DO NOT commit localization PRs to the `dev-1.26` branch.

/hold

**Comment by sftim** [comment]
Here's my concern: the release docs team have to merge in changes and resolve conflicts. We don't expect those teams to include representatives from every launched localization. It's much easier to have that team specialize in the upstream (English) localization.

If any localization team wants to make a head start on changes, those are OK to work on, but PRs will need to rebase after the v1.26 release and should be draft before then.

**Comment by iyear** [comment]
@divya-mohan0209 @tengqm @sftim Thank you very much for your reviews and replies, I didn't read the contribution files carefully enough to cause the conflict. It's true that I shouldn't commit to `dev-1.26` and cause trouble for the team.

And what should I do next? It looks like merging this PR cause more trouble.

**Pull request closed without merging**